packages/console/src/App.tsx

```diff
@@ -23,14 +23,12 @@ import Main from './pages/Main';
 void initI18n();
 
 const Content = () => {
-  const {
-    tenants: { data, isSettle },
-  } = useContext(TenantsContext);
+  const { tenants, isSettle } = useContext(TenantsContext);
   const currentTenantId = getUserTenantId();
 
   const resources = deduplicate([
     ...(currentTenantId && [getManagementApi(currentTenantId).indicator]),
-    ...(data ?? []).map(({ id }) => getManagementApi(id).indicator),
+    ...(tenants ?? []).map(({ id }) => getManagementApi(id).indicator),
```

Comment:
It seems that this `...(currentTenantId && [getManagementApi(currentTenantId).indicator])` is redundant, since `...(tenants ?? []).map(({ id }) => getManagementApi(id).indicator),` covers it.

Reply:
this is by design which will save one redirect in test case 3:

user has multiple tenants including foo, access http://localhost:3003/foo to sign in, shows foo's admin console